src/config.js

```diff
@@ -54,7 +54,10 @@ export const DB_CERT = env('DB_CERT', undefined);  // path to a certificate (cer
 export const PC_URL = env('PC_URL', 'https://apps.pathwaycommons.org/');
 export const REACH_URL = env('REACH_URL', 'http://reach.baderlab.org/api/uploadFile');
 export const BIOPAX_CONVERTER_URL = env('BIOPAX_CONVERTER_URL', 'https://biopax.baderlab.org/convert/v2/');
-export const GROUNDING_SEARCH_BASE_URL = env('GROUNDING_SEARCH_BASE_URL', 'https://grounding.baderlab.org');
+//
+// TODO PUT THIS BACK TO THE PROPER VALUE -- master only for convenient testing
+export const GROUNDING_SEARCH_BASE_URL = env('GROUNDING_SEARCH_BASE_URL', 'https://master.grounding.baderlab.org');
```

Comment:
This should be changed back before merging

Reply:
@maxkfranz I did so and comitted . However, the build has been failed so I have not been merged this PR yet. The build error logs are as the following:

```
Failed to install 10. Remote repository may not be reachable.
Using locally available version 10, if applicable.
$ nvm use 10
N/A: version "10 -> N/A" is not yet installed.
You need to run "nvm install 10" to install it before using it.
Unable to use 10
The command "false" failed and exited with 1 during .
Your build has been stopped.
```